module/VuFindAdmin/src/VuFindAdmin/Controller/TagsController.php

```diff
@@ -198,18 +198,18 @@ protected function getConfirmDeleteMessages($count)
         $user = $this->getTable('user')
```

Comment:
I'd like an explicit null check for getParam return value for each type. Currently, in the unlikely case that the object selected gets deleted before the form is submitted, the current code results in "All", which is not right. I think the code should first check getParam(), and if it's set, either fetch the data or set the message to '<unknown>' or something.

Later on, when adding the 'tag_delete_filter' message, I'd rather check for message strings other than 'All' rather than comparing them with false.